internal/services/mssql/mssql_server_resource.go

```diff
@@ -517,13 +541,29 @@ func resourceMsSqlServerUpdate(d *pluginsdk.ResourceData, meta interface{}) erro
 		return fmt.Errorf("updating Connection Policy for %s: %+v", id, err)
 	}
 
+	vaState := sqlvulnerabilityassessmentssettings.SqlVulnerabilityAssessmentStateDisabled
+	if d.Get("express_vulnerability_assessment_enabled").(bool) {
+		vaState = sqlvulnerabilityassessmentssettings.SqlVulnerabilityAssessmentStateEnabled
+	}
+
+	va := sqlvulnerabilityassessmentssettings.SqlVulnerabilityAssessment{
+		Properties: &sqlvulnerabilityassessmentssettings.SqlVulnerabilityAssessmentPolicyProperties{
+			State: &vaState,
+		},
+	}
+
+	if _, err := vaClient.CreateOrUpdate(ctx, *id, va); err != nil {
+		return fmt.Errorf("updating Express Vulnerability Assessment settings for %s: %+v", *id, err)
+	}
+
```

Comment:
Should we wrap this in a `d.HasChanges` check? Written this way, `ignore_changes` will be ignored